references/mosaic-architecture.md

```
@@ -0,0 +1,267 @@
+      {...rest}
+      css={styles({ intent, size, disabled })(theme)}
+    />
+  );
+});
+```
+
+## Coexistence with existing system
+
+### Provider tree
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)